stable/kube-slack/templates/deployment.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apps/v1beta1
+apiVersion: {{ .Values.deployment.apiVersion }}
```

Comment:
why not just set `apps/v1`?

Reply:
In case Someone is Running Eks 1.13,1.14 or 1.15

Reply:
For EKS 1.16 this Value needs to be overwritten. But if you like i can specificy it automatically based on EKS Server version @cpanato

Reply:
Determined automatically @cpanato